package-rfcx/rfcx/_api_rfcx.py

```diff
@@ -25,8 +25,14 @@ def tags(token, type, labels, start, end, sites, limit):
     path = '/v2/tags'
     url = '{}{}?{}'.format(host, path, urllib.parse.urlencode(data, True))
     return _request(url, token=token)
```

Comment:
Please remove the old endpoints from the code.

Reply:
I have create a task for tags to detection next sprint: https://jira.rfcx.org/browse/CS-407